package.json

```diff
@@ -33,7 +33,8 @@
     "chalk": "~1.1.1",
     "underscore.string": "~3.2.2",
     "yeoman-generator": "~0.20.3",
-    "yo": "~1.4.8"
+    "yo": "~1.4.8",
+    "mkdirp": "~0.5.1"
```

Comment:
Why to we need mkdirp and not just use the built in file system functions?

http://yeoman.io/authoring/file-system.html

Also, if we do need mkdirp, it needs to be in alphabetical order. Easiest way is to use `npm install mkdirp --save`.